Specifications/Language/3_Expressions/ValueLiterals.md

```diff
@@ -119,35 +121,34 @@ All items in the created array are set to the [default value](#default-values) o
 Arrays containing qubits or callables must be properly initialized with non-default values before their elements may be safely used.
 Suitable initialization routines can be found in the `Microsoft.Quantum.Arrays` namespace.
 
-## Tuple Literals
+## Tuple literals
 
-A [tuple](https://github.com/microsoft/qsharp-language/tree/main/Specifications/Language/4_TypeSystem#available-types) literal is a sequence of one or more expressions of any type, separated by commas, enclosed in `(` and `)`. The type of the tuple includes the information about each item type.
+A [tuple](https://github.com/microsoft/qsharp-language/tree/main/Specifications/Language/4_TypeSystem#available-types) literal is a sequence of one or more expressions of any type, separated by commas and enclosed in parentheses `(` and `)`. The type of the tuple includes the information about each item type.
 
 | Value Literal | Type |
 | --- | --- |
 | `("Id", 0, 1.)` | `(String, Int, Double)` |
 | `(PauliX,(3,1))` | `(Pauli, (Int, Int))` |
 
-Tuples containing a single item are treated as identical to the item itself, both in type and value. We refer to this a [singleton tuple equivalence](https://github.com/microsoft/qsharp-language/blob/main/Specifications/Language/4_TypeSystem/SingletonTupleEquivalence.md#singleton-tuple-equivalence).
+Tuples containing a single item are treated as identical to the item itself, both in type and value, and are called a [singleton tuple equivalence](https://github.com/microsoft/qsharp-language/blob/main/Specifications/Language/4_TypeSystem/SingletonTupleEquivalence.md#singleton-tuple-equivalence).
```

Comment:
I think this was a typo in the original copy - "a singleton tuple equivalence" should've been "as singleton tuple equivalence."

```suggestion
Tuples containing a single item are treated as identical to the item itself, both in type and value, which is called [singleton tuple equivalence](https://github.com/microsoft/qsharp-language/blob/main/Specifications/Language/4_TypeSystem/SingletonTupleEquivalence.md#singleton-tuple-equivalence).
```